core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -3405,6 +3405,341 @@ class KafkaApisTest {
     assertEquals(Errors.GROUP_AUTHORIZATION_FAILED, response.error)
   }
 
+  @ParameterizedTest
```

Comment:
I added a few unit tests here.

Reply:
Do we have a test here for the fetchAllOffsets path?

Reply:
Yes.

Reply:
```
val groups = Map(
        "group-1" -> List(
          new TopicPartition("foo", 0),
          new TopicPartition("foo", 1)
        ).asJava,
        "group-2" -> null,
        "group-3" -> null,
      ).asJava
```

with `null`, it fetch all offsets.

Reply:
Ah I missed this. Thanks for sharing.